<ui version="4.0" >
 <class>PicSearchOptions</class>
 <widget class="QDialog" name="PicSearchOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Search Images</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>10</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>5</number>
     </property>
     <item row="1" column="0" >
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Start at:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2" >
      <widget class="QLineEdit" name="fileEdit" />
     </item>
     <item row="1" column="2" >
      <widget class="QPushButton" name="changeDirButton" >
       <property name="text" >
        <string>Change...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="directoryEdit" >
       <property name="minimumSize" >
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="textLabel1_2" >
       <property name="text" >
        <string>Search for:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>5</number>
     </property>
     <item>
      <widget class="QCheckBox" name="caseInsensitiveCheck" >
       <property name="text" >
        <string>Case insensitive search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="recursiveSearch" >
       <property name="text" >
        <string>Search recursively</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>81</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="searchLabel" >
       <property name="text" >
        <string>Searching</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Q3ProgressBar" name="progressBar1" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>250</width>
         <height>22</height>
        </size>
       </property>
       <property name="totalSteps" >
        <number>100</number>
       </property>
       <property name="progress" >
        <number>0</number>
       </property>
       <property name="centerIndicator" >
        <bool>false</bool>
       </property>
       <property name="percentageVisible" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startButton" >
       <property name="text" >
        <string>Start Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ProgressBar</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3ProgressBar</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
